tt_metal/tools/profiler/process_device_log.py

```diff
@@ -243,6 +244,11 @@ def import_device_profile_log(logPath):
                     devicesData["devices"][chipID] = {
                         "cores": {core: {"riscs": {risc: {"timeseries": [(timerID, timeData, statData)]}}}}
                     }
+                if doOpsDetection and "zone_name" in timerID.keys() and "dispatch" in timerID["zone_name"].lower():
+                    logger.warning(
+                        "Dispatch core data is detected, stats across ops are skipped as op detection cannot work with dispatch data present"
```

Comment:
hard to understand what it means. Could you add more explanations(something actionable).

Reply:
How is "Dispatch core data is detected: stats across ops are skipped as op detection cannot work with dispatch data present. Please unset the TT_METAL_DEVICE_PROFILER_DISPATCH environment variable"